distribution/docs/src/main/resources/draft-documentation.adoc

```diff
@@ -0,0 +1,39 @@
+= ${branding-expanded} Documentation
+include::${project.build.directory}/doc-contents/_contents/config.adoc[]
+
+= Introduction
+
+== About ${branding}
+include::{adoc-include}/_introduction/introduction-contents.adoc[]
+
+include::{adoc-include}/_introduction/applications-contents.adoc[]
+
+== Documentation Guide
+
+include::{adoc-include}/_introduction/documentation-notes-contents.adoc[]
+
+<<<
+
+= Core Concepts
```

Comment:
that's a good distinction. I still think an intro to "what a catalog is" is a good starting point, then ingest, then content